ci-operator/config/openshift-priv/ovn-kubernetes/openshift-priv-ovn-kubernetes-master.yaml

```diff
@@ -407,6 +407,14 @@ tests:
     env:
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-vsphere
+- always_run: false
+  as: e2e-azure-ovn-virt
```

Comment:
Please add the `techpreview` suffix to the jobs names, for example:
`e2e-azure-ovn-virt-techpreview`